### PR TITLE
Add Lab GCS

### PR DESCRIPTION
**Pull request by ThomasVrancken**

### CODE REVIEW

File: labs/09_gcs/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+2. :sunglasses: Note that you can give access to other users to your GCP project through the [IAM page](https://console.cloud.google.com/iam-admin/) - so multiple project can work on one project **with the same billing account**
+3. Make sure to have a [billing account](https://console.cloud.google.com/billing/) (remember that you can request credits to the teaching staff!)
+
+Enable the Cloud Storage API:
+1. Navigate to the [Cloud storage tab](https://console.cloud.google.com/storage)
+2. If requested Enable the API
+
+And we will then **create a bucket** in which we will store the files for this lab.
+
+You can do so from the UI directly.
```
Comment (Mapirlet) [inline labs/09_gcs/README.md:26]: What about all the possible options that can be chosen (e.g. region : single region, etc.) ?

File: labs/09_gcs/README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+
+## 1. Using the python SDK to upload files
+
+Create a jupyter notebook (e.g. `cloud_storage.ipynb`) and do the following:
+
+First, install the Google Cloud Storage Python client library:
+
+```sh
+!pip install google-cloud-storage
```
Comment (Mapirlet) [inline labs/09_gcs/README.md:46]: Maybe it's just me, but I find it strange to put it in the jupyter notebook and not ask them to run it in the terminal unless it's a Colab notebook.

File: labs/09_gcs/README.md (hunk 1 above)
Comment (ThomasVrancken) [reply labs/09_gcs/README.md:26]: Will make sure to voice that different regions and availability can be picked

File: labs/09_gcs/README.md (hunk 2 above)
Comment (ThomasVrancken) [reply labs/09_gcs/README.md:46]: Well it's a way of doing it. As I want the students to see clearly they need that installed and I don't want to bother with getting venv on a notebook kernel I think this is fine

**Pull request merged**